tests/test_qml/test_qmlbot.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+
+import pytest
+
+
+def test_load_from_string_wrong_syntax(qmlbot):
+    qml = "import QtQuick 2.0 Rectangle{"
+    with pytest.raises(RuntimeError):
+        qmlbot.loads(qml)
+
+
+def test_load_from_string(qmlbot):
+    text = "that's a template!"
+    qml = (
```

Comment:
Minor, but you can use `textwrap.dedent` to align this with the test.

Reply:
oh thanks